src/app/auth/services/user.service.ts

```diff
@@ -23,6 +23,12 @@ export class UserService {
       {},
     );
   }
+  public resendConfirmationEmail(email: string): Observable<any> {
+    return this.httpClient.put<any>(
+      this.url + 'user/resend-confirm-email/' + email,
```

Comment:
the email should be added automatically from the backend side. Since this endpoint should require to be logged in and thus checking if the user is verified and if not resending that email